### DISCUSSION TITLE
Tooltip

### DISCUSSION THREAD

**Discussion by emergenitro** [EssentialsX/Essentials#3862] [Ideas and Feature Suggestions] [answered]
So, my idea isn't that big a suggestion however, isn't it possible to have a tooltip when you hover over a player name that shows the player's balance (essentials economy obviously) and other things such as rank and other things.

It should be customisable and should placeholders and stuff. Just thought that this ought to be a good idea.

**Comment by mdcfe** [comment, marked as answer]
Even once EssentialsX adds support for chat components (#2391), this feels a bit out-of-scope for EssentialsX. I wouldn't totally reject this if someone wanted to implement it, but I also don't feel that this is a high priority for us, as I'd imagine other dedicated chat plugins can already do this.
